### PR TITLE
fix(core): Columns should be customise in Workflow

### LINKED ISSUE #41: Whenever we click on When this happens 3 options displays they should be customizable
**Describe the bug**
Workflow: Create WF>>When this happens>>3 options displays>> rename the On interval to "Every time Period"

for further information 
https://sourcefuse.atlassian.net/browse/OPX-8996

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by Deepika516**
resolved customise column issue of workflow

GH-41

## Description
made columns custimisable so that i want to change the name of the column i can 

Fixes #41 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by AryanshSourcefuse — APPROVED** [review]

File: projects/workflows-creator/src/lib/builder/builder.component.ts [hunk 1]
```diff
@@ -60,8 +58,9 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
     private readonly elements: ElementService<E>,
     private readonly cdr: ChangeDetectorRef,
     private readonly localizationSvc: LocalizationProviderService,
-  ) {}
-
+  ) {
+    this.localizationSvc.setLocalizedStrings(this.localizedStringMap);
```
Comment (akshatdubeysf) [inline projects/workflows-creator/src/lib/builder/builder.component.ts:62]: arre why? you are doing this in onInit na?

File: projects/workflows-creator/src/lib/builder/builder.component.ts [hunk 2]
```diff
@@ -149,21 +142,23 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
...
     this.cdr.detectChanges();
   }
-
   /**
    * > If the diagram and state have changed, restore the diagram and state from the builder
    * @param {SimpleChanges} changes - SimpleChanges - the changes that have occurred in the component
    */
   async ngOnChanges(changes: SimpleChanges) {
+    if (changes['localizedStringMap'] && this.localizedStringMap) {
+      this.localizationSvc.setLocalizedStrings(this.localizedStringMap);
```
Comment (akshatdubeysf) [inline projects/workflows-creator/src/lib/builder/builder.component.ts:153]: why do you need this line if is already the first line of initiatenode

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: projects/workflows-creator/src/lib/builder/builder.component.ts (hunk 1 above)
Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/builder.component.ts:62]: sir its in Constructer, I had to remove this from ngonint n put in into the constructer bcos it was not working as expected , the Default strings were redendered on UI but not the localized one.

File: projects/workflows-creator/src/lib/builder/builder.component.ts (hunk 2 above)
Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/builder.component.ts:153]: If user changes any localized string after Init Thats y we code here in ngOnChanges

Comment (Deepika516) [reply projects/workflows-creator/src/lib/builder/builder.component.ts:153]: removed.

**Review by akshatdubeysf — APPROVED** [review]

**Pull request merged**